### PR TITLE
Feat : Cart summary Added

### PR DESCRIPTION
**Pull request by t007rushi**
✅Quantity update In card
✅Card Summary on add, delete, increment, decrement from Cart Page

recording

<details><summary>Browser metadata</summary>

```
Path:      /cart
Browser:   Chrome 99.0.4844.82 on Windows 10
Viewport:  1536 x 775 @1.25x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-11--game-kart-ecom.netlify.app%2Fcart&os=Windows&os_version=10&browser=Chrome&browser_version=99.0&resolution=1536x775&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-11--game-kart-ecom.netlify.app/cart) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=623f5aa2cbf4f20009d6886c&commentID=623f5e4382a2fb655c6374ef&resolution=resolved)

### CODE REVIEW

**Review by omgaikwad — APPROVED** [review]
The code is clean and well written! You are good to go!

File: src/Context/cart-context.js [hunk 1]
```diff
@@ -43,12 +45,25 @@ const CartProvider = ({ children }) => {
     }
   };
 
+  //QUANTIY INCR/DECR
+  const updateCartQuant = async (id, quantAct) => {
+    const data = await updateCartQuantService(id, user, quantAct);
+    // console.log(data)
+    if (data !== undefined) {
+      setCart(data.cart);
+    }
```
Comment (JantuDeb) [inline src/Context/cart-context.js:54]:
```
if(!data) setCart(data.cart);
```

**Review by JantuDeb — APPROVED** [review]
Well structured code

**Review by horsemaker — APPROVED** [review]
Everything looks perfect here! 👌

File: src/Context/cart-context.js (hunk 1 above)
Comment (t007rushi) [reply src/Context/cart-context.js:54]: oh yes only if there

**Pull request merged**